include/seqan3/alphabet/detail/member_exposure.hpp

```diff
@@ -103,14 +103,14 @@ constexpr alphabet_type & assign_rank(alphabet_type & alph, underlying_rank_t<al
  * \details
  * Use this e.g. to newly create alphabet letters from rank:
  * ~~~{.cpp}
- * auto l = assign_rank(dna5{}, 1);  // l is of type dna5 and == 'C'_dna5
+ * auto l = assign_rank_to(1, dna5{});  // l is of type dna5 and == 'C'_dna5
```

Comment:
Can you do a `\snippet test/snippet/alphabet/adaptation/uint.cpp assign_rank` here instead?

Reply:
this will all go away / be completely changed with the next PRs anyway